### PR TITLE
Use normal RAPIDS environment for CI

### PR DESCRIPTION
**Pull request by jakirkham**
Previously we had some workarounds here to get the bleeding edge CuPy 9 package, GCC 9.3.0 compilers, etc. However now that RAPIDS supports these, these customizations can be removed. This should get things in better alignment with RAPIDS generally.

cc @raydouglass @gigony @quasiben

### CODE REVIEW

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -79,21 +79,10 @@ gpuci_conda_retry build -c ${LIBCUCIM_BLD_PATH} -c conda-forge/label/cupy_rc -c
 gpuci_logger "Install cuCIM and its dependencies"
 
 gpuci_logger "Install dependencies"
-gpuci_conda_retry create -n cucim -y -c conda-forge -c conda-forge/label/cupy_rc -c rapidsai-nightly \
+gpuci_conda_retry install -y -c conda-forge -c rapidsai-nightly \
     rapids-doc-env \
```
Comment (jakirkham) [inline ci/gpu/build.sh:83]: Not sure if we still need to explicitly install this or if it is already included

File: ci/gpu/build.sh [hunk 2]
```diff
@@ -79,21 +79,10 @@ gpuci_conda_retry build -c ${LIBCUCIM_BLD_PATH} -c conda-forge/label/cupy_rc -c
...
-    flake8 \
-    pytest \
-    pytest-cov \
-    python=${PYTHON_VER} \
-    conda-forge/label/cupy_rc::cupy=9 \
-    cudatoolkit=${CUDA_VER} \
-    numpy \
-    scipy \
-    scikit-image=0.18.1 \
     openslide
```
Comment (jakirkham) [inline ci/gpu/build.sh:84]: Submitted PR ( https://github.com/rapidsai/integration/pull/263 ) to add this to the integration package. Can remove here if we decide it is ok to add over there

Comment (jakirkham) [reply ci/gpu/build.sh]: This PR has been merged and this code dropped. Marking resolved

**Review by ajschmidt8 — APPROVED** [review]

**Comment by jakirkham** [comment]
rerun tests

File: conda/recipes/cucim/meta.yaml [hunk 3]
```diff
@@ -20,23 +20,25 @@ build:
...
+    - sysroot_linux-64 2.17
   host:
     - cudatoolkit {{ cuda_version }}.*
     - python {{ python_version }}.*
     - libcucim {{ version }}.*
     - click
-    - conda-forge/label/cupy_rc::cupy=9
-    - numpy
+    - cupy 9.*
+    - numpy >=1.17.0
```
Comment (jakirkham) [inline conda/recipes/cucim/meta.yaml:32]: Was needed as NumPy 1.16 was being pulled in, which is incompatible with RAPIDS and CuPy 9 and caused solver conflicts. Forcing 1.17 fixed the issue

**Review by ajschmidt8 — APPROVED** [review]

**Review by quasiben — APPROVED** [review]

**Comment by quasiben** [comment]
Thanks @jakirkham !!!

**Review by ajschmidt8 — APPROVED** [review]

**Review by gigony — APPROVED** [review]
Thanks @jakirkham ! 
Looks good to me.

**Comment by jakirkham** [comment]
@gpucibot merge

**Pull request merged**